cppcache/integration-test/testThinClientSecurityAuthorization.cpp

```diff
@@ -54,13 +54,15 @@ const std::string locHostPort =
 std::shared_ptr<CredentialGenerator> credentialGeneratorHandler;
 
 std::string getXmlPath() {
-  char xmlPath[1000] = {'\0'};
-  const char *path = std::getenv("TESTSRC");
-  ASSERT(path != nullptr,
+  std::string path = std::string(std::getenv("TESTSRC"));
```

Comment:
Same comment as above regarding code safety for std::getenv

Reply:
N/A. Test removed.